templates/k3s/overlay/opt/corral/k3s/init-cluster.sh

```diff
@@ -1,24 +1,23 @@
 #!/bin/bash
-set -ex
 
-CORRAL_api_host="kube-api.${CORRAL_fqdn}"
+CORRAL_api_host="${CORRAL_fqdn}"
 echo "corral_set api_host=${CORRAL_api_host}"
 
-mkdir -p /etc/rancher/k3s
-cat > /etc/rancher/k3s/config.yaml <<- EOF
+config="write-kubeconfig-mode: 644
 cluster-init: true
 tls-san:
   - ${CORRAL_api_host}
-EOF
+"
 
-apt install -y jq
-
-curl -sfL https://get.k3s.io | KUBERNETES_VERSION=${CORRAL_kubernetes_version} sh -
+mkdir -p /etc/rancher/k3s
+cat > /etc/rancher/k3s/config.yaml <<-EOF
+${config}
+EOF
 
-sed -i "s/127.0.0.1/kube-api.${CORRAL_fqdn}/g" /etc/rancher/k3s/k3s.yaml
+curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=${CORRAL_kubernetes_version} sh -
```

Comment:
Does this mean the nodes will always have internet access? I'm assuming Airgap is not in scope in this PR.

Reply:
Yes, I didn't add airgap scope with this PR. I prioritized the K3s embedded ETCD HA since we don't have it